google-cloud-spanner/src/main/java/com/google/cloud/spanner/GrpcStruct.java

```diff
@@ -184,6 +188,9 @@ private Object writeReplace() {
             case DATE:
               builder.set(fieldName).toDateArray((Iterable<Date>) value);
```

Comment:
nit:  indentation looking different for the switch statements cases

Reply:
How is it different?

Reply:
Just realised these are part of  case ARRAY scope . IN the compact mode it was showing in different indentation. Can be resolved.